pkg/antctl/raw/supportbundle/command.go

```diff
@@ -104,7 +107,8 @@ func init() {
 		Command.Flags().StringVarP(&option.dir, "dir", "d", "", "support bundles output dir, the path will be created if it doesn't exist")
 		Command.Flags().StringVarP(&option.labelSelector, "label-selector", "l", "", "selector (label query) to filter Nodes for agent bundles, supports '=', '==', and '!='.(e.g. -l key1=value1,key2=value2)")
 		Command.Flags().BoolVar(&option.controllerOnly, "controller-only", false, "only collect the support bundle of Antrea controller")
-		Command.Flags().StringVarP(&option.nodeListFile, "node-list-file", "f", "", "only collect the support bundle of Antrea controller")
+		Command.Flags().StringVarP(&option.nodeListFile, "node-list-file", "f", "", "only collect the support bundle of specific nodes filtered by names in a file (one node name per line)")
+		Command.Flags().Uint32Var(&option.days, "days", 0, "if > 0, only collect logs that have been generated in the preceding specified number of days")
```

Comment:
what is the behavior if it is --days 0? maybe we should specifically state that if setting days, it must be a positive int.

Reply:
If days is not greater than 0, it will take no effect, maybe the help string need a little tweak to make it more clear. About  the timestamp filter, I dont't get any strong opinion on either one.

Reply:
Yestday I talked to some folks in the team, seems `kubectl logs --since` is a good example of how to do this. I will post the command line help string in here
```bash
 --since=0s: Only return logs newer than a relative duration like 5s, 2m, or 3h. Defaults to all logs. Only one of
since-time / since may be used.
      --since-time='': Only return logs after a specific date (RFC3339). Defaults to all logs. Only one of since-time /
since may be used.
```
So i was thinking we can do something similar to the `--since` args.